pagopa/io-app
# fix: [IOBP-836] Remove assistance for FCI into payments zendesk modal

## Short description
This PR removes the FCI information asked when users open a support request from the payments screen.

## List of changes proposed in this pull request
- Edited the flag to open a zendesk ticket request setting to `false` the `requestForFci` property
- Setted to true the value to show data of the payments in the zendesk permissions request modal

## How to test
- Start any payment from the payments screen
- Add an invalid notice code
- Tap on the "Request assistance" button 


## Before
<img src="https://github.com/user-attachments/assets/edd4aa68-5735-4df7-af08-34b62571817e" width="300" />
